cli/QuickstepCli.cpp

```diff
@@ -150,6 +154,14 @@ DEFINE_bool(initialize_db, false, "If true, initialize a database.");
 DEFINE_bool(print_query, false,
             "Print each input query statement. This is useful when running a "
             "large number of queries in a batch.");
+DEFINE_string(profile_file_name, "",
```

Comment:
One thing which we can do is generate a default profile name if none is specified. The default profile name can be something like "quickstep-profile-<timestamp>.prof", so that name collisions are avoided. That way, we can get rid of the check ``!quickstep::FLAGS_profile_file_name.empty()`` in Line 450. It's ok if we don't do it right away, in which case you can leave a TODO note.